packages/main/src/plugin/dockerode/libpod-dockerode.ts

```diff
@@ -72,9 +72,35 @@ export interface PlayKubeInfo {
   Volumes: { Name: string }[];
 }
 
+export interface PodCreatePortOptions {
+  host_ip: string;
+  container_port: number;
+  host_port: number;
+  protocol: string;
+  range: number;
+}
+
+export interface PodCreateOptions {
+  name: string;
+  portmappings?: PodCreatePortOptions[];
+}
+
+export interface ContainerCreateOptions {
+  command?: string[];
+  entrypoint?: string | string[];
+  env?: { [key: string]: string };
+  pod?: string;
+  hostname?: string;
+  image?: string;
+  name?: string;
+}
+
 // API of libpod that we want to expose on our side
 export interface LibPod {
+  createPod(podOptions: PodCreateOptions): Promise<{ Id: string }>;
+  createContainer(containerCreateOptions: ContainerCreateOptions): Promise<{ Id: string; Warnings: string[] }>;
   listPods(): Promise<PodInfo[]>;
+  //getContainerInspect(podId: string): Promise<ContainerInspectInfo>;
```

Comment:
unintentional comment out / should be removed?

Reply:
comment is removed
(sorry it has been using an amend instead of a fixup)